solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -208,7 +214,7 @@ const fetchBondingData = async (web3Context) => {
       }
     }
 
-    for (const [operatorAddress, isWithdrawable] of operators.entries()) {
+    for (const [operatorAddress, managedGrantAddress] of operators.entries()) {
```

Comment:
I think, that `isWithdrawable` might be still needed to handle the visibility of the button.